src/app/layout.tsx

```
@@ -36,6 +37,24 @@ export default function RootLayout({
             </AuthInterceptor>
           </TokenRefresher>
         </Providers>
+
+        <Script
+          id="sleekplan-widget"
+          strategy="afterInteractive"
+          dangerouslySetInnerHTML={{
+            __html: `
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)